samples/azure/azure-service-fabric-routing/sample.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Azure Service Fabric Partition Aware Routing
+reviewed: 2017-02-14
+component: Core
+related:
+- nservicebus/azure-service-bus
+---
+
+
+## Prerequisites
+
+1. Azure Service Fabric [dev cluster](https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-get-started) running configured with 5 nodes.
+2. An Azure Service Bus namespace that can be used for communication between the instances.
+3. A machine level environment variable called "AzureServiceBus.ConnectionString" that contains the manage connectionstring to the Azure Service Bus namespace.
+
+## Scenario
+
+The scenario used in this sample covers a voting system. In this voting system the casted votes are counted by candidate. Next to this the system also counts the total number of votes casted in each zip code. When voting is over, the system will publish the results and report them on the Service Fabric diagnostics infrastructure.
+
+For sake of simplicity, there are only 2 candidates in the election, called "John" and "Abby". Zip codes are always assumed to be valid integers with a length up to 5 digits in the range (00000 to 99000).
+
+
+## Solution structure
+
+The solution contains the following projects:
+
+ * Contracts: This project contains class that are shared by the other projects. These classes include message definitions as well as partition aware routing definitions that will be plugged into the NServiceBus pipeline.
+ * CandidateVoteCount: This Service Fabric service contains the logic to count the votes by candidate while the votes come in. It also sends these votes to the `ZipCodeVoteCount` endpoint for tracking by zipcode and it will report the results when the voting period is over.
```

Comment:
> It also sends these votes to the `ZipCodeVoteCount` endpoint

How about `It sends `ZipCodeVoteCount` command ... `

> and it will report the results when the voting period is over.

How about `When the voting period ends it reports the voting results